packages/PriceLockup/PriceLockup.md

```diff
@@ -1,27 +1,24 @@
 ### Usage Criteria
 
+- `PriceLockup` is available in three different sizes:
+  - Use `small` for pricing in product catalogue pages (not in cards)
+  - Use `medium` for pricing in product compare cards
+  - Use `large` for pricing in banners and promo cards
+- The size prop refers to the size of the PriceLockup rather than the viewport it is being used in
+- PriceLockup is responsive depending on the screensize beinv viewed at
```

Comment:
Typo! "beinv" should be "being". Maybe change it to "screen size it is being viewed at" to make the sentence flow a bit better.